7Cav-Alive.Altis/scripts/missionEnd/missionEnd_StrongPoint.sqf

```diff
@@ -0,0 +1,45 @@
+/* This script will:
+ *
+ * Define some parameters, log which box was activated, Initialize some variables, remove the initial addaction, 
+ * and then perform some user feedback. The user feedback is achieved by animating the box, completing a task,
+ * then changing the orbat markers on the map.
+ *
+ * After the user feedback, we check if all the boxes were activated. If they were, we end the mission. If they
+ * weren't, we do nothing.
+ *
+ */
+
+//Addaction return values for the boxes, so we can get the global variable
+params ["_target", "_caller", "_actionId", "_arguments"];
+
+//Set the local variable equal to the global variable
+private _boxNumber = missionNameSpace getVariable "endMissionNumber";
+
+//log which box was activated
+diag_log formatText["%1 has been activated!", _target];
+
+//Increase the local variable by 1
+_boxNumber = _boxNumber + 1;
+ 
+//Remove all the actions on the box
+removeAllActions _target;
+
+//Animate the box to keyframe 3
+[_target,3] call BIS_fnc_DataTerminalAnimate;
+
+//Succeed the task 
+["Defend The Satellite!", "SUCCEEDED"] call BIS_fnc_taskSetState;
+
+//Orbat : Switch statement for the different boxes
+switch (_target) do {

```

Comment:
Not 100% sure on this one, but I think you'll have an error here. `_target` is an object while `"Term1"` is a string, and I think Arma will get upset with you if you compare them.